webapp/i18n/en_AU.json

```diff
@@ -0,0 +1,250 @@
+{
+  "BoardComponent.add-a-group": "+ Add a group",
```

Comment:
It seems to add a lot of translation overhead to include a complete translation of `en_AU` instead of just the keys that are different and let the code use `en` for keys that are not found.

Reply:
@wiggin77 I'm not sure Weblate supports diffing them like that. I'll look into it